NightscoutUploadKit/DeviceStatus/PredictedBG.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PredictedBG.swift
+//  RileyLink
+//
+//  Created by Pete Schwamb on 8/8/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+public struct PredictedBG {
+    let values: [Double]
+    let cob: [Double]?
+    let iob: [Double]?
+
+    public init(values: [Double], cob: [Double]? = nil, iob: [Double]? = nil) {
```

Comment:
That was for current IOB/COB; I haven't added the forecasted values to the PR yet, as I just realized what they were, today, while talking to @jasoncalabrese